src/%IPM/Lifecycle/Base.cls

```diff
@@ -492,8 +492,7 @@ Method %Reload(ByRef pParams) As %Status
 			Set tSC = $$$ERROR($$$GeneralError,"module.xml is malformed.")
 			Quit
 		}
-		set first = $$$lcase(first)
-    If ($Piece(first,".",*) '= "zpm") {
+    	If ($Piece(first,".",*) '= "ZPM") {
```

Comment:
return back lcase

Reply:
Hi Dmitry! There is an on-going discussion about upper case and lower case! So please hold before we all reach a final agreement. Thank you!

Reply:
This is not a part of that discussion at all, here is just normalization for content of module.xml
with this change, it will not support AnyModule.zpm, only AnyModule.ZPM will be allowed.

for instance
https://github.com/intersystems-community/restforms-angular/blob/master/module.xml